examples/features/ports/startup-hook.sh

```diff
@@ -1,4 +1,8 @@
 #!/bin/bash
 
 pip install 'pydantic<2.0.0' # resolves 'pydantic.dataclasses.dataclass only supports init=False'
-pip install -U "ray[air]"
+pip install -U "ray[air]==2.9.3"
+pip install 'pandas==1.5.3'
+pip show pandas
+pip show pydantic
+pip show ray
```

Comment:
```suggestion
```

Reply:
ops that triggered another ci cycle. automerge ftw